Packs/FeedUnit42v2/ReleaseNotes/1_0_50.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Unit 42 ATOMs Feed
+
+- Updated the Docker image to: *demisto/taxii2:1.0.0.90938*.
+- Added "rawJSON" key for the attack pattern indicator type.
+- Added "rawJSON" key for the report indicator type.
+- Added "rawJSON" key for the indicator type.
```

Comment:
is something missing here? Which indicator type?

Reply:
@sapirshuker ?